target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -285,6 +285,7 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan:1" "3:lan:2" "4:lan:3" "5:lan:4" "1:wan"
 		;;
+	belkin,f9j1108-v2|\
```

Comment:
Please move the entire block to maintain alphabetic sorting based on the new first entry.